src/vos/tests/evt_ctl.c

```diff
@@ -1078,36 +1201,42 @@ ts_cmd_run(char opc, char *args)
 {
 	int	rc;
 
+	tst_fn_val.optval = args;
```

Comment:
Another comment unrelated to your patch.   The variable args is set from optarg.    I just noticed this due to your changes.   It's rather confusing naming.  It's really just the single option argument string from the command line.   You don't need to change it but if you're so inclined while fixing the linting issues, I won't complain :-)